scripts/generate_search.py

```diff
@@ -124,22 +124,50 @@ def index_dir(dirname):
 
 index_dir(base_dir)
 
-# {
-# 	"data": [
-# 		{
-# 			"title": "Select Statement",
-# 			"text": "The SELECT statement retrieves rows from the database",
-# 			"category": "docs",
-# 			"url": "../docs/sql/statements/select"
-# 		},
-# 		{
-# 			"title": "Update Statement",
-# 			"text": "The UPDATE statement modifies the values of rows in a table.",
-# 			"category": "docs",
-# 			"url": "../docs/sql/statements/update.md"
-# 		}
-# 	]
-# }
+# extract functions
+def extract_markdown_text(text):
+	parse_node = marko.parse(text)
+	return extract_text(parse_node)
+
+def sanitize_function(text):
+	return text.replace(' , ', ', ').replace('( ', '(').replace(' )', ')').replace("'", '')
+
+def sanitize_desc(text):
+	return text.replace(' .', '.')
+
+function_list = {}
+
+def extract_functions(text, full_path):
+	functions = re.findall('\n[|]([^|\n]+)[|]([^|\n]+)[|]([^|\n]+)[|]([^|\n]+)[|]', text)
```

Comment:
```suggestion
	functions = re.findall(r'\n[|]([^|\n]+)[|]([^|\n]+)[|]([^|\n]+)[|]([^|\n]+)[|]', text)
```

Regex should always be escaped as a raw (r) string in python. I'd also recommend replacing the `[|]` parts with `\|` once that's done